docs/medical-api/api-reference/facility/update-facility.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Update Facility"
-description: "Creates a Facility in Metriport where your patients receive care."
+description: "Updates a Facility in Metriport where your patients receive care."
```

Comment:
let's also update https://docs.metriport.com/devices-api/more-info/our-integrations specify `Yes` for WH in withings